packages/block-editor/src/components/block-toolbar/style.scss

```diff
@@ -57,6 +57,25 @@
 		}
 	}
 
+	&.is-contentonly-mode {
```

Comment:
Can we use is-dark-toolbar class to be more explicit. There might be different modes that would want to trigger that behavior.

Reply:
@youknowriad i have updated the classname.